docs/_scripts/meetup_events.py

```diff
@@ -23,18 +25,40 @@ def load_meetups():
 
 client = meetup.api.Client()
 
-for m in meetups:
+for meetup in meetups:
     # Substring the meetup.com and trailing slash. EWWW
     #print(m[23:-1])
 
-    t = client.GetEvents(group_urlname=m[23:-1])
+    t = client.GetEvents(group_urlname=meetup[23:-1])
+
+    for event in t.results:
+        # pprint('##'+event['group']['name'])
+        # pprint(event['name'])
+        # pprint(event['event_url'])
+
+        ##################################################
+        ms_since_the_epoch = event['time']
+        month = strftime("%B", gmtime(ms_since_the_epoch))
+        day = strftime("%d", gmtime(ms_since_the_epoch))
+        date = month + ' ' + day
+
+        if 'venue' in event:
+            venue = event['venue']
+            pprint(date + ' - ' + venue['city'])
+        else:
+            pprint("LOOOOOOOOOK!!!!!!!")
+
+            # this prints "March 10 - Australi" ... why???
```

Comment:
You're truncating the string with `[15:-1]` , or did I misunderstand the question?

Reply:
This is correct! We'll forgo this truncation for now, then (it's not very important). ☺️

Reply:
Brain is finally warming up ... I realized that `[15:]` was what I wanted, not `[15:-1]`. 😛